tests/phpunit/includes/abstract-testcase.php

```diff
@@ -142,10 +142,10 @@ public function set_up() {
 	/**
 	 * Sets the bcrypt cost option for password hashing during tests.
 	 *
```

Comment:
```suggestion
	 *
	 * @since 6.9.0 $algorithm can only be a string.
	 *
```

Noting that this is a change that could break implementions. I think it's fine, but should get called out.

Reply:
This code is in the tests so shouldn't be a problem.

Reply:
Ah, good point.  Disregard this suggestion.